plugin/binlog_utils_udf/binlog_utils_udf.cc

```diff
@@ -168,6 +168,21 @@ const char *get_short_binlog_name(const std::string &binlog_name) noexcept {
   return binlog_name.c_str() + dirname_length(binlog_name.c_str());
 }
 
+std::pair<int, std::list<std::string>> get_normalized_log_index() {
```

Comment:
Function comment would be good to have. we convert a binlogfilename without the "./" (ie binlog.00001) to "./binlog.00001" ?

Reply:
Done